tests/phpunit/tests/functions/mceSetDirection.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Tests for _mce_set_direction function.
+ *
+ * @group functions.php
+ *
+ * @covers ::_mce_set_direction
+ */#
+class Tests_Functions_MceSetDirection extends WP_UnitTestCase {
+	/**
+	 * @var WP_Locale
+	 */
+	protected $locale;
+
+	public function set_up() {
+		parent::set_up();
+		$this->locale = new WP_Locale();
+	}
+
+	/**
+	 * @ticket 57192
+	 */
+	public function test__mce_set_direction() {
```

Comment:
```suggestion
	public function test_mce_set_direction() {
```
A double `_` is not convention in the test suite, even for `_private` functions.